code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -744,3 +744,7 @@ emp_act
 		to_chat(src, "<span class='danger'>Your [head.name] protects you from the [hot ? "hot" : "cold"] liquid!</span>")
 		return FALSE
 	return TRUE
+
+/mob/living/carbon/human/projectile_hit_check(obj/item/projectile/P)
+	return (HAS_TRAIT(src, TRAIT_FLOORED) && !density) // hit mobs that are intentionally lying down to prevent combat crawling.
```

Comment:
```suggestion
	return HAS_TRAIT(src, TRAIT_FLOORED) && !density // hit mobs that are intentionally lying down to prevent combat crawling.
```
Don't need the brackets